New article for season of giving 2020 (#60)
SADevs/sadevs.github.io: content/articles/SADevs-season-of-giving-2020.md

Title: SA Devs Season of Giving
Date: 11-01-2020
Tags: season-of-giving, donations, charity

> Slug: season-of-gicing-2020

Reviewer: ```suggestion
Slug: season-of-giving-2020
```

During the month of November, let's work together as a group to give to those who can use some help. Donate to your 
favorite charity (we've got some suggestions below), submit your receipt to SA Devs, and we'll amplify your donation. At
the end of the month, the SA Devs org will match up to $2000 of community donations to the SA Food Bank.

Want to sponsor some matching funds? You can sponsor a day, a week, or the entire month and any amount/total. Reach out
to the admins and we'll add your matching and notify you of totals!

We also understand some people might want to donate in other ways and we want to celebrate that too! If you donate your 
time, canned goods, or do any other charitable work and you'd like it celebrated as part of SA Devs Season of Giving,
reach out to the admins and we'll add it to our donation list! 

### Suggested Charities
These are charities that we feel are doing good work in our community. Don't feel like you have to donate to these 
organizations, but if you're looking for a place to donate these are good choices!

Do you have an organization you like to work with? Reach out to the admins and we'll add it to our list!

#### San Antonio Food Bank

> https://safoodbank.org/

Reviewer: ```suggestion
<https://safoodbank.org/>
```

#### Hill Country Daily Bread

> https://hillcountrydailybread.com/

Reviewer: ```suggestion
<https://hillcountrydailybread.com/>
```

####Get Up Community Center

> https://www.getupcommunity.org/food-pantry

Reviewer: ```suggestion
<https://www.getupcommunity.org/food-pantry>
```

### How to record a donation

1. Take a screenshot or PDF of your donation receipt

> + Open a DM to @sadevbot

Reviewer: ```suggestion
+ Open a DM to `@sadevbot`
```

> + Send ./donation {amount} --make-public and attach your receipt to the message 

Reviewer: ```suggestion
+ Send `./donation {amount} --make-public` and attach your receipt to the message 
```

If you want to keep your donation private (the admins will still see your name/info but we won't put it on the site) 
leave out --make-public

Make sure your amount matches the amount on your receipt. You can enter it as $##.## i.e. $20.00 (or even just $20).
+ Sadevbot will send your donation on to the admins for review and it'll get added to the website (just below here in 
the donations list). Private donations will just list an amount. 

> It'll also update totals in #sadevs-season-of-giving-2020

Reviewer: ```suggestion
It will also update totals in `#sadevs-season-of-giving-2020`
```
